tests/test_cli.py

```diff
@@ -235,6 +235,15 @@ def test_create_bdev_ana_ipv6(self, caplog, gateway):
 
     def test_create_subsystem_ana(self, caplog, gateway):
         caplog.clear()
+        with pytest.raises(Exception) as ex:
```

Comment:
@gbregman 🖖 why this addition is required? could you explain?

Reply:
@baum this is new code added by Leonid. But he didn't add a test scenario to test it. So I added it with my changes.